dashboard/static/src/__tests__/App.test.js

```diff
@@ -0,0 +1,10 @@
+import { shallow } from 'enzyme';
+import React from "react";
+import App from '../js/App';
```

Comment:
Out of scope for this review, but I need your help to refactor the directory structure of the project to make it easier to deploy.

[I created an issue where we can discuss it further](https://github.com/uwcirg/stayhome-dashboard/issues/13)